cpp/tests/io/csv/csv_test.cu

```diff
@@ -457,3 +463,37 @@ TEST(gdf_csv_test, Dates)
 								   1149638400000, 1126828800000, 2764800000) );
 	}
 }
+
+TEST(gdf_csv_test, FloatingPoint)
+{
+	const char* fname			= "/tmp/CsvFloatingPoint.csv";
+	const char* names[]			= { "A" };
+	const char* types[]			= { "float32" };
+
+	std::ofstream outfile(fname, std::ofstream::out);
+	outfile << "5.6;0.5679e2;1.2e10;0.07e1;3000e-3;";
```

Comment:
if 12.34e0 is a valid format, I think you should include it in the test since it' a special case in the code above.

Reply:
Added to both pytest and gtest, thanks for the suggestion!